pkg/network/tracer/tracer_linux_test.go

```diff
@@ -2031,3 +2032,20 @@ func testConfig() *config.Config {
 	}
 	return cfg
 }
+
+func (s *TracerSuite) TestOffsetGuessIPv6DisabledCentOS() {
+	t := s.T()
+	cfg := testConfig()
+	// disable IPv6 via config to trigger logic in GuessSocketSK
+	cfg.CollectTCPv6Conns = false
+	cfg.CollectUDPv6Conns = false
+	kv, err := kernel.HostVersion()
+	kv470 := kernel.VersionCode(4, 7, 0)
+	require.NoError(t, err)
+	if kv >= kv470 {
+		// will only be run on kernels < 4.7.0 matching the GuessSocketSK check
+		t.Skip()
```

Comment:
You can also pass the message to `Skip` so it is printed when `-v` is used for running tests.